### PR TITLE
Updated Liquid

### PR DESCRIPTION
**Pull request by Seggan**
## Description
Updated Liquid entry

## Reason
Liquid is now incompatible with earlier releases

### CODE REVIEW

File: resources/repos.json [hunk 1]
```diff
@@ -815,10 +815,10 @@
 				"1": "1.15.x, 1.16.x"
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: No 1.14 support?

File: resources/repos.json [hunk 2]
```diff
@@ -834,10 +834,10 @@
...
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: No 1.14 support?

File: resources/repos.json [hunk 3]
```diff
@@ -769,7 +769,7 @@
 		},
 		"dependencies": {
 			"Minecraft Version(s)": {
-				"1": "1.15.x, 1.16.x"
+				"1": "all"
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: Help please, I tested it on Minecraft Beta 1.4 and it doesn't work!!!! pls fix

Comment (Seggan) [reply resources/repos.json]: argh

File: resources/repos.json [hunk 4]
```diff
@@ -769,7 +769,7 @@
 		},
 		"dependencies": {
 			"Minecraft Version(s)": {
-				"1": "1.15.x, 1.16.x"
+				"1": "1.14+"
```
Comment (TheBusyBiscuit) [inline resources/repos.json]: Did you test it on MC 1.42 :LUL:
No but seriously, please stick to the current format of listing versions and only list the versions you actually know it works on.

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**